coffeelint.json

```diff
@@ -1,103 +0,0 @@
-{
```

Comment:
This should be updated to use our eslint rules - PR is huge, sorry if it's already done, just making a note in case it's not and I forget before I get to the end :p